lib/flappy_search_bar.dart

```diff
@@ -405,12 +406,45 @@ class _SearchBarState<T> extends State<SearchBar<T>>
                     duration: Duration(milliseconds: _animate ? 1000 : 0),
                     child: AnimatedContainer(
                       duration: Duration(milliseconds: 200),
-                      width:
-                          _animate ? MediaQuery.of(context).size.width * .2 : 0,
+                      width: () {
+                        // Get the width of the cancellation widget.
+                        final RenderBox renderBox = _cancellationWidgetKey
+                            .currentContext
+                            .findRenderObject();
+                        final double cancellationWidgetWidth =
+                            renderBox.size.width;
+
+                        double toBeReturned;
+
+                        if (_animate) {
+                          final fullWidth = MediaQuery.of(context).size.width;
```

Comment:
You have just forgotten the type : `final double fullWidth`

Reply:
woops silly mistake